packages/sources/react/src/components/VtmnLink/types.ts

```diff
@@ -0,0 +1 @@
+export type VtmnLinkSize = 'small' | 'medium' | 'large';
```

Comment:
I would be tented to make an enumeration here. To avoid setting size to 'small' but using VtmnLinkSize.SMALL instead.

Reply:
I made a request change (by accident) but it is more a suggestion ;)

Reply:
Same comment  as in #276

Reply:
For propTypes like this I prefer union type (https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types)
Because that's the point, we need to know what are the different types possible and I used to see that on React components.